postgresql-orioledb/benchmark.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# digest: sha256:3304142dbe8de8d5bbaa0e398cca58683ed603add4524c3582debf9c119994f1
+VERSION=beta12-pg17
+CONTAINER_NAME=orioledb-clickbench
+
+MODE="$1"
+if [[ -z "$MODE" ]]
+then
+  MODE=default-config
+fi
+
+if [[ ! $MODE =~ ^(default-config|tuned-config)$ ]]; then
+   echo "Unknown mode: '$MODE'. Expected one of 'default-config', 'tuned-config'"
+   exit 1
+fi
+
+echo "Chosen mode: $MODE"
+
+echo "Installing dependencies..."
+sudo apt-get update -y
+sudo apt-get install -y docker.io pigz postgresql-client
+
+echo "Starting OrioleDB Docker container with name $CONTAINER_NAME. Using tag $VERSION..."
+# Increase shared memory size, because Docker default will hit the limit ("ERROR:  could not resize shared memory segment")
+MEM_SIZE=$(grep MemTotal /proc/meminfo | awk '{print $2}')
+SHM_SIZE=$(echo "$MEM_SIZE/2/1024" | bc)
+mkdir -p /tmp/data
+sudo docker run --name $CONTAINER_NAME -v /tmp/data:/tmp/data --shm-size="$SHM_SIZE"m -p 5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust -d orioledb/orioledb:$VERSION
+
+# Similar (but not identical) to PostgreSQL configuration
+if [ "$MODE" = "tuned-config" ]; then
+  echo "Updating configuration"
+  THREADS=$(nproc)
+  CPUS=$(($THREADS / 2))
+
+  # Since we are only using OrioleDB tables, set to 1/4 of RAM and keep default value for shared buffers
+  # See https://www.orioledb.com/docs/usage/configuration#orioledbmain_buffers
+  MAIN_BUFFERS=$(($MEM_SIZE / 4))
+  EFFECTIVE_CACHE_SIZE=$(($MEM_SIZE - ($MEM_SIZE / 4)))
+  MAX_WORKER_PROCESSES=$(($THREADS + 15))
+
+  envsubst <<EOF | sudo docker exec -i $CONTAINER_NAME sh -c 'tee -a /etc/postgresql/postgresql.conf'
```

Comment:
To keep simple things simple, I suggest to
- remove the "tuned" config,
- move special settings (l. 44-51) into the default config.

I am asking because 1) in a perfect world, no database tuning (e.g. settings) is needed at all 2) it generally makes more sense to distinguish default configurations and configurations with tuned indexes.